### PR TITLE
gh-87286: Add a number of LOG_* constants to syslog

### PR DESCRIPTION
**Pull request by ronaldoussoren**
This adds a number of syslog facilities to the syslogmodule.c.
These values are available on macOS.

* Issue: gh-87286

### CODE REVIEW

File: Doc/library/syslog.rst [hunk 1]
```diff
@@ -90,7 +90,12 @@ Facilities:
    :const:`LOG_AUTH`, :const:`LOG_LPR`, :const:`LOG_NEWS`, :const:`LOG_UUCP`,
    :const:`LOG_CRON`, :const:`LOG_SYSLOG`, :const:`LOG_LOCAL0` to
    :const:`LOG_LOCAL7`, and, if defined in ``<syslog.h>``,
-   :const:`LOG_AUTHPRIV`.
+   :const:`LOG_AUTHPRIV`, :const:`LOG_FTP`, :const:`LOG_NETINFO`, 
+   :const:`LOG_REMOTEAUTH`, :const:`LOG_INSTALL`, :const:`LOG_RAS`.
+
+   .. versionchanged:: 3.10
+       Added :const:`LOG_NETINFO`, :const:`LOG_REMOTEAUTH`, :const:`LOG_INSTALL`,
```
Comment (akulakov) [inline Doc/library/syslog.rst]: also LOG_FTP

File: Misc/NEWS.d/next/Library/2021-02-03-21-26-57.bpo-43120.4fbCBl.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Added :const:`LOG_NETINFO`, :const:`LOG_REMOTEAUTH`, :const:`LOG_INSTALL`,
```
Comment (akulakov) [inline Misc/NEWS.d/next/Library/2021-02-03-21-26-57.bpo-43120.4fbCBl.rst]: also LOG_FTP

File: Doc/library/syslog.rst [hunk 3]
```diff
@@ -90,7 +90,12 @@ Facilities:
    :const:`LOG_AUTH`, :const:`LOG_LPR`, :const:`LOG_NEWS`, :const:`LOG_UUCP`,
    :const:`LOG_CRON`, :const:`LOG_SYSLOG`, :const:`LOG_LOCAL0` to
    :const:`LOG_LOCAL7`, and, if defined in ``<syslog.h>``,
-   :const:`LOG_AUTHPRIV`.
+   :const:`LOG_AUTHPRIV`, :const:`LOG_FTP`, :const:`LOG_NETINFO`, 
+   :const:`LOG_REMOTEAUTH`, :const:`LOG_INSTALL`, :const:`LOG_RAS`.
+
+   .. versionchanged:: 3.10
```
Comment (akulakov) [inline Doc/library/syslog.rst]: Should this be 3.11 now?

**Review by akulakov — COMMENTED** [review]
By the way I also was not able to find some of these levels in `man 3 syslog` on my MacOS, (for example LOG_NETINFO, LOG_RAS, etc - but it's old, 10.12, so were they added more recently? I couldn't find them online either.

File: Doc/library/syslog.rst [hunk 4]
```diff
@@ -107,22 +107,62 @@ The module defines the following functions:
...
+.. data:: LOG_EMERG
+          LOG_ALERT
+          LOG_CRIT
+          LOG_ERR
+          LOG_WARNING
+          LOG_NOTICE
+          LOG_INFO
+          LOG_DEBUG
+
+   Priority levels (high to low):
```
Comment (hugovk) [inline Doc/library/syslog.rst]: This applies to the preceding list:
```suggestion
   Priority levels (high to low).
```

Comment (serhiy-storchaka) [inline Doc/library/syslog.rst]: Should not it be above the previous list?

File: Doc/library/syslog.rst [hunk 5]
```diff
@@ -107,22 +107,62 @@ The module defines the following functions:
...
+
+
+.. data:: LOG_AUTH
+          LOG_AUTHPRIV
+          LOG_CRON
+          LOG_DAEMON
+          LOG_FTP
```
Comment (hugovk) [inline Doc/library/syslog.rst]: Lint nit:
```suggestion
          LOG_FTP
```

File: Doc/library/syslog.rst (hunk 4 above)
Comment (hugovk) [reply Doc/library/syslog.rst]: Could be, currently it renders like this:

image

Where this text is the ["description details"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd) of the last "description term":

image

File: Doc/library/syslog.rst [hunk 6]
```diff
@@ -107,22 +107,62 @@ The module defines the following functions:
...
+          LOG_LOCAL0
+          LOG_LOCAL1
+          LOG_LOCAL2
+          LOG_LOCAL3
+          LOG_LOCAL4
+          LOG_LOCAL5
+          LOG_LOCAL6
+          LOG_LOCAL7
+
+   Facilities, depending on availabiltiy in ``<syslog.h>`` for :const:`LOG_AUTHPRIV`,
```
Comment (AlexWaygood) [inline Doc/library/syslog.rst]: Typo :)

```suggestion
   Facilities, depending on availability in ``<syslog.h>`` for :const:`LOG_AUTHPRIV`,
```

Comment (ronaldoussoren) [reply Doc/library/syslog.rst]: Spelling is hard. Thanks!

File: Misc/NEWS.d/next/Library/2023-12-07-16-55-41.gh-issue-87286.MILC9_.rst [hunk 7]
```diff
@@ -0,0 +1,3 @@
+Added :const:`LOG_FTP`, :const:`LOG_NETINFO`, :const:`LOG_REMOTEAUTH`,
+:const:`LOG_INSTALL`, :const:`LOG_RAS`, and :const:`LOG_LAUNCHD` tot the
+:mod:`syslog` module, all of them constants on used on macOS.
```
Comment (terryjreedy) [inline Misc/NEWS.d/next/Library/2023-12-07-16-55-41.gh-issue-87286.MILC9_.rst:3]: I believe 'tot' should be 'to' and 'constants on used on' should be 'constants used on'.

**Pull request merged**